runtimes/php/php-apps.md

```diff
@@ -250,26 +250,12 @@ You can find more documentation about composer configuration at [getcomposer.com
 
 You use Artisan to manage your project and you want to execute _artisan migrate_ before running your app.
 
-First, add a file `ccbuild.sh` at the root of your project with these lines:
+To do this, we use a post build hook, you have to set a new environment variable on your Clever application as following
```

Comment:
@KannarFr Can you fix this issue and then I can merge this. Feel free to add another commit which reverts those changes and then I will squash & merge.